xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/displayers/DisplayerHtml.vue

```diff
@@ -31,22 +31,31 @@
     all the displayer default behavior
   -->
   <BaseDisplayer
-    class="displayer-html"
-    :property-id="propertyId"
-    :entry="entry"
+      class="displayer-html"
+      :property-id="propertyId"
+      :entry="entry"
+      :is-view.sync="isView"
   >
 
     <!-- Provide the Html Viewer widget to the `viewer` slot -->
     <template #viewer>
-        <div
-            class="html-wrapper"
-            v-html="value"
-        ></div>
+      <div
+          class="html-wrapper"
+          v-html="value"
+      ></div>
     </template>
 
     <!-- Provide the Html Editor widget to the `editor` slot -->
-    <!-- TODO: implemtent the edit widget fetch from the server -->
-    <template #editor></template>
+    <!-- TODO: implement the edit widget fetch from the server -->
+    <template #editor>
+      <div v-html="editField"
+           tabindex="0"
+           v-autofocus
+           @focusout="applyEdit($event.target.value)"
+           @keypress.enter="applyEdit($event.target.value)"
+           @keydown.esc="cancelEdit">
+      </div>
```

Comment:
the focus management is currently poorly handled, the focus is something lost when interacting with the edit form labels, and something clicking outside of the cell does not focus out.